analytic_engine/src/instance/alter.rs

```diff
@@ -29,6 +29,20 @@ use crate::{
     table_options,
 };
 
+/// Policy of how to perform flush operation.
+#[derive(Default, Debug, Clone, Copy)]
+pub enum TableAlterSchemaPolicy {
+    /// Unknown flush policy, this is the default value.
+    #[default]
+    Unknown,
```

Comment:
`Unknown` policy seems weird and unnecessary, and what's the difference between `Unknown` and `Noop`?

Reply:
There are serval points
- I want each operation has its own policy, each policy name has meaning. E.g.: flush has `Purge` and `Dump` and alter has `Noop`.
- The only common policy variant is `Unknown`. And all operations report error when encount `Unknown` policy.
- The policy can only be set by `RoleTable`, other place can only set it to `Unknown` via `Default::default()`.

So the `Unknown` policy is a guard variant to avoid a operation has not checked by `RoleTable`. But the only problem I have is I cannot prevent others from constructing un-Default policy, maybe this need to be enforced via a runtime check.

Reply:
Understood. How about giving some comments to the `UnKnown` Policy, especially for:
> And all operations report error when encount Unknown policy.

Reply:
> Understood. How about giving some comments to the `Unknown` Policy, especially for:
> 
> > And all operations report error when encount Unknown policy.

Ok 👌